src/azure-cli/azure/cli/command_modules/acr/_validators.py

```diff
@@ -78,3 +79,10 @@ def validate_task_argument(string, is_secret):
             return {'type': 'Argument', 'name': comps[0], 'value': local_value, 'isSecret': is_secret}
         return {'type': 'Argument', 'name': comps[0], 'value': '', 'isSecret': is_secret}
     return None
+
+
+def validate_retention_days(namespace):
+    days = namespace.days
+    if isinstance(days, int):
+        if days and (days < 0 or days > 365):
+            raise CLIError("invalid value for days: should be from 0 to 365")
```

Comment:
Invalid

Reply:
nit: no strong opinion here but prefer leaving the validation to the server side if possible.

Reply:
RP will add the validation in parallel. Don't want to block the release for that or not have the validation.